components/Header.jsx

```diff
@@ -50,96 +121,231 @@ const useStyles = makeStyles(theme => ({
       textDecoration: 'underline',
     },
   },
+  imageStyle: {
+    verticalAlign: 'middle',
+  },
 }));
 
 const activeStyle = {
-  borderBottom: `1px solid ${colors.primaryFocus}`,
+  borderBottom: `2px solid ${colors.primaryFocus}`,
 };
 
 // TODO: links/routing, mobile
-function Header() {
+const Header = () => {
   const classes = useStyles();
-  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [isDashboardOpen, setIsDashboardOpen] = React.useState(false);
+  const [isIconOpen, setIsIconOpen] = React.useState(false);
+
+  const handleDashboardClick = event => {
+    setIsDashboardOpen(event.currentTarget);
+  };
+
+  const handleDashboardClose = () => {
+    setIsDashboardOpen(null);
+  };
 
-  const handleClick = event => {
-    setAnchorEl(event.currentTarget);
+  const handleIconClick = event => {
+    setIsIconOpen(event.currentTarget);
   };
 
-  const handleClose = () => {
-    setAnchorEl(null);
+  const handleIconClose = () => {
+    setIsIconOpen(null);
   };
 
   return (
     <AppBar position="static" className={classes.appBar}>
       <Toolbar>
-        <Typography variant="h4" className={classes.title}>
-          <Link to="/" className={classes.link}>
-            311DATA
-          </Link>
+        <Typography variant="h4" className={classes.textLogo}>
+          <Link to="/" className={classes.link}>311DATA</Link>
         </Typography>
         <NavLink className={classes.link} to="/map" style={({ isActive }) => (isActive ? activeStyle : null)}>
           <Button className={classes.button}>Map</Button>
         </NavLink>
-        <Button
+        {/* <Button
```

Comment:
I actually think we're ok to completely remove this. When Dashboard gets reintroduced, it's unlikely we'll keep the implementation that we're seeing here